src/netlink.c

```diff
@@ -543,6 +569,8 @@ static int qos_filter_cb(const struct nlmsghdr *nlh, void *args) {
 
       submit_one(dev, "ipt_bytes", type_instance, ts->bytes);
       submit_one(dev, "ipt_packets", type_instance, ts->packets);
+      submit_one(dev, "ipt_drops", type_instance, ts->drops);
+      submit_one(dev, "ipt_overlimits", type_instance, ts->overlimits);
```

Comment:
@octo Same as above, I assume not every qdisc has the concept of overlimits, in HFSC this _should be_ the amount of time/packets it steals from sibling classes. Drops are packets that are gone, but drops might only occur in leaf nodes in some setups.